librz/type/helpers.c

```diff
@@ -101,9 +101,6 @@ RZ_API RZ_OWN RzType *rz_type_pointer_of_base_type_str(const RzTypeDB *typedb, R
  */
 RZ_API RZ_OWN RzType *rz_type_pointer_of_type(const RzTypeDB *typedb, RZ_NONNULL RzType *type, bool is_const) {
 	rz_return_val_if_fail(typedb && type, NULL);
-	if (type->kind == RZ_TYPE_KIND_IDENTIFIER) {
-		return rz_type_pointer_of_base_type_str(typedb, type->identifier.name, is_const);
-	}
 	RzType *newtype = RZ_NEW0(RzType);
```

Comment:
Was this unused?

Reply:
Probably needs clarification. I added this to the commit description:

    rz_type_pointer_of_type() is now also always creating a pointer type of
    the given type. Before, it would have failed when the pointee was an
    identifier that did not exist in the database. But this may be a valid
    case e.g. while still building the database from DWARF and such
    sanitization goes beyond what one would expect from
    rz_type_pointer_of_type().